Jenkinsfile

```diff
@@ -890,7 +890,7 @@ pipeline {
                         job_step_update(
                             functionalTest(
                                 inst_repos: daosRepos(),
-                                inst_rpms: functionalPackages(1, next_version, 'tests-internal'),
+                                inst_rpms: functionalPackages(1, next_version, '{tests-internal,devel}'),
```

Comment:
If ftest needs pydaos, and pydaos needs devel, then I would think the tests rpm should require devel in the spec to avoid silent dependencies.

Reply:
Good point.

pydaos is installed in daos-client but only for python3.6 and even then not for virtual environments.  avocado on 3.6 will only install in a virtual environment (or at least with our CI images) as it conflicts with the disto PyYAML package so avocado cannot use the daos-client version of pydaos and hence ftest requires daos-devel.  Once this PR is in I'd like to look at moving forward the avocado version and for that we need to bump the python version used so at that point we'll have to start using virtual environments anyway but that's for another PR.

The one part of this PR that I could split our would be the pydaos packaging which is really DAOS-14849.  It's not clear where I'd be able to draw the line with splitting out the requirements.txt files in that case but it would be possible although this getting this far wasn't easy.

Are you OK with adding a dependency from daos-tests to daos-devel?

Reply:
> Are you OK with adding a dependency from daos-tests to daos-devel?

This work requires that, right? It's just not written in the spec currently. So I think we _**have to**_. I'm okay with it, and can't think of a reason not to.

FWIW, we seem to ship daos-devel with signed RPMs, but not daos-tests. So if someone goes out of their way to get daos-tests, they certainly should have daos-devel on hand

Reply:
Done.  That wasn't too tricky in the end.

Reply:
Since ftest uses gdb, I think that makes sense